### PR TITLE
Squished table headers fix.

### PR DESCRIPTION
**Pull request by jbarnicle**
Changed table layout to auto to prevent table headers from being squished. Made the main content div scrollable horizontally to accommodate larger tables.

### CODE REVIEW

File: atf_eregs/static/regulations/css/less/typography.less [hunk 1]
```diff
@@ -392,6 +392,10 @@ The `<ol>` tag needs to have the appropriate list type applied
 ```
 */
 
+.main-content {
+    overflow-x:scroll;
+}
```
Comment (cmc333333) [inline atf_eregs/static/regulations/css/less/typography.less]: What does this look like for very long paragraphs? Does the text wrap or does the page scroll?

**Comment by cmc333333** [comment]
Ideally this type of change would be inherited from -site. More fire for the "make it modular" gods.

File: atf_eregs/static/regulations/css/less/typography.less (hunk 1 above)
Comment (jbarnicle) [inline atf_eregs/static/regulations/css/less/typography.less]: Paragraphs don't force an overflow - the text continues to wrap in the same manner.

Comment (cmc333333) [inline atf_eregs/static/regulations/css/less/typography.less]: :+1: do you think it makes sense to add this to -site, too?

Comment (jbarnicle) [inline atf_eregs/static/regulations/css/less/typography.less]: I think so - I will make that change so it is in the next push.  In upcoming versions I think it should be namespaced a little better to ensure it only applies to ereg-based tables / content.

**Pull request merged**